### ISSUE TITLE
Duplicate lines in Patches from DCS 2.7.10.19473

### ISSUE THREAD

**Issue by BlueFinBima** [HeliosVirtualCockpit/Helios#579]
Patch problems introduced by https://github.com/HeliosVirtualCockpit/Helios/commit/7126631cd01686bb364f92cfd904b1a885c6cd28 **fresh patch cluster for DCS 2.7.10.19473**.
Basically there are duplicate lines in several of the lua files and from a quick scan, I think the list is ....  
```
Patching/Patches/DCS/002_007_00010_19473_00000/Viewports/Mods/aircraft/F14/Cockpit/Scripts/HeadUpDisplay/Indicator/init.lua.gpatch
Patching/Patches/DCS/002_007_00010_19473_00000/Viewports/Mods/aircraft/F14/Cockpit/Scripts/HeadUpDisplay/Indicator/init.lua.grevert
Patching/Patches/DCS/002_007_00010_19473_00000/Viewports/Mods/aircraft/F14/Cockpit/Scripts/RWR/Indicator/Pilot/init.lua.gpatch
Patching/Patches/DCS/002_007_00010_19473_00000/Viewports/Mods/aircraft/F14/Cockpit/Scripts/RWR/Indicator/Pilot/init.lua.grevert
Patching/Patches/DCS/002_007_00010_19473_00000/Viewports/Mods/aircraft/F14/Cockpit/Scripts/RWR/Indicator/RIO/init.lua.gpatch
Patching/Patches/DCS/002_007_00010_19473_00000/Viewports/Mods/aircraft/F14/Cockpit/Scripts/RWR/Indicator/RIO/init.lua.grevert
Patching/Patches/DCS/002_007_00010_19473_00000/Viewports/Mods/aircraft/FA-18C/Cockpit/Scripts/Multipurpose_Display_Group/HUD_AVQ32/indicator/AVQ32_init.lua.gpatch
Patching/Patches/DCS/002_007_00010_19473_00000/Viewports/Mods/aircraft/FA-18C/Cockpit/Scripts/Multipurpose_Display_Group/HUD_AVQ32/indicator/AVQ32_init.lua.grevert
Patching/Patches/DCS/002_007_00010_19473_00000/Viewports/Mods/aircraft/FA-18C/Cockpit/Scripts/TEWS/indicator/RWR_ALR67_init.lua.gpatch
Patching/Patches/DCS/002_007_00010_19473_00000/Viewports/Mods/aircraft/FA-18C/Cockpit/Scripts/TEWS/indicator/RWR_ALR67_init.lua.grevert
```

**Comment by BlueFinBima** [comment]
Not certain that there is an adequate way to address this problem because fixing the patches would mean that previously applied patches could not be removed. 

Commit in #581 may still have problems the F18 RWR_ALR67_init and  AVQ32_init.lua but I can't recreated at the moment.

**Comment by BlueFinBima** [comment]
Fixed in [1.6.5305.2 &  3](https://github.com/HeliosVirtualCockpit/Helios/releases/latest)
